Snakefile

```diff
@@ -44,7 +44,7 @@ rule all:
 
 rule process_sample_one_file_in_sample:
     container:
-        "docker.io/reanahub/reana-demo-agc-cms-ttbar-coffea:1.0.0"
+        "registry.cern.ch/docker.io/reanahub/reana-demo-agc-cms-ttbar-coffea:1.0.0"
```

Comment:
We cannot easily use `registry.cern.ch` because then people would not be able to run this example outside of CERN. Let's keep `docker.io/reanahub/reana-demo-agc-cms-ttbar-coffea:1.0.0`.

(An alternative would be to push all our images to CERN Harbor and you that registry, but that's something we could do centrally later for all the examples.)